### PR TITLE
Fixed some method names in ControlNetVersioner. 

### PR DESCRIPTION
**Pull request by kberryUSGS**
Also added IsJigsawRejecte to ControlPoint.

### CODE REVIEW

**Comment by kberryUSGS** [comment]
[Update Write() & toPvl() code to use the ControlPoint and ControlMeasure methods we actually added... and add some missing ones.](https://trello.com/c/WLv5FSIC/131-update-write-topvl-code-to-use-the-controlpoint-and-controlmeasure-methods-we-actually-added-and-add-some-missing-ones)

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 1]
```diff
@@ -1933,19 +1929,17 @@ namespace Isis {
 
       protoPoint.set_aprioriradiussource(controlPoint->GetAprioriRadiusSource());
 
-      // FIXME: None of Apriori(X,Y,Z) is available directly from ControlPoint in the API
-      if (controlPoint->HasAprioriRadiusSourcefile()) { // DNE
+      if (controlPoint->HasAprioriRadiusSourcefile()) {
```
Comment (AgoinsUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Typo, the 'f' in HasAprioriRadiusSourceFile() should be capitalized.

Comment (kberryUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: :+1: Thanks for catching this.

**Comment by kberryUSGS** [comment]
Okay, typo fixed.

**Pull request merged**